assets/models.py

```diff
@@ -49,6 +49,20 @@ def get_base_queryset(self):
 
 class Asset(ModelChangeMixin, models.Model):
 
+    def audit_compare(self, field, old, new):
+        """
+        Overridden as the base implementation doesn't correctly detect differences for
+        MultiSelectField fields.
+
+        :param field: The field being compared
+        :param old: the original value
+        :param new: the updated value
+        :return: whether or not a change has been detected
+        """
+        if new is not None and isinstance(field, MultiSelectField):
+            return old != list(new)
```

Comment:
Would this not be better as
```python
# MultiSelectField are essentially sets so we consider "old" to be equal to "new"
# if the symmetric difference is the empty set.
return len(set(old) ^ set(new)) == 0
```
which would have the advantage of being insensitive to ordering.